astropy/utils/masked/tests/test_masked.py

```diff
@@ -17,6 +17,7 @@
 from astropy.utils.compat import NUMPY_LT_2_0
 from astropy.utils.compat.optional_deps import HAS_PLT
 from astropy.utils.masked import Masked, MaskedNDArray
+from astropy.visualization.wcsaxes.utils import MATPLOTLIB_LT_3_8
```

Comment:
Oops, this caused a problem: now the `astropy.utils.masked` tests get skipped if one does not have matplotlib installed... I'll make a fix...

Reply:
fix in #16843

Reply:
Ugh. Sorry for the oversight. I merged your fix. Thanks!